packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -118,6 +120,7 @@ export default async (homeP, endowments) => {
       usdcDenom,
     },
     feeConfig: parseFeeConfigArgs(),
+    feedPolicy: JSON.parse(feedPolicy),
```

Comment:
This will crash if no `--feedPolicy` arg is given.

Not the end of the world, but "--feedPolicy is required" would be a nicer diagnostic than "cannot parse JSON"

Reply:
Doesn't this make the a3p-integration test fail?

Oh... #10517 hasn't landed yet. sigh.

Reply:
Happy to wait for that to go in, your call though.

Reply:
Added a check for `feedPolicy` also

Reply:
#10517 landed.

It has an `--oracleSets` flag. I suggest renaming that to `--net`. Then add a `feedPolicy` to each of the `configurations`.